blueprints/ember-bootstrap/index.js

```diff
@@ -1,241 +1,333 @@
+    let stylesPath = join('app', 'styles');
+    let filePath = join(stylesPath, `app.${extension}`);
 
-    this.ui.writeLine(chalk.green(`Installing for Bootstrap ${bootstrapVersion} using preprocessor ${preprocessor}`));
+    // Create styles folder if not exists
+    if (!fs.existsSync(stylesPath)) {
+      fs.mkdirSync(stylesPath);
+    }
 
-    this.bootstrapVersion = bootstrapVersion;
-    this.preprocessor = preprocessor;
-  },
+    let actionMessage = fs.existsSync(filePath)
+      ? `Added import statement to ${filePath}`
+      : `Created ${filePath}`;
+    this.ui.writeLine(chalk.green(actionMessage));
 
-  afterInstall() {
-    return this.adjustBootstrapDependencies()
-      .then(() => this.adjustPreprocessorDependencies())
-      .then(() => this.addPreprocessorStyleImport())
-      .then(() => this.addBuildConfiguration());
+    return this.insertIntoFile(filePath, importStatement);
   },
 
-  removePackageFromBowerJSON(dependency) {
-    this.ui.writeLine(chalk.green(`  uninstall bower package ${chalk.white(dependency)}`));
-    return new rsvp.Promise(function(resolve, reject) {
-      try {
-        let bowerJSONPath = 'bower.json';
-        let bowerJSON = fs.readJsonSync(bowerJSONPath);
-
-        delete bowerJSON.dependencies[dependency];
+  writeBuildConfig() {
+    let config = normalizeConfig(this.addonConfig);
+    if (Object.keys(config).length === 0) {
+      return;
+    }
+    let configBuilder = this.configBuilder;
+    let ui = this.ui;
+
+    if (configBuilder) {
+      configBuilder.set(GEN_NAME, '{}');
+      configBuilder.setProperties(stringifyConfig(config));
+      return configBuilder
+        .save()
+        .then(() =>
+          ui.writeLine(
+            chalk.green(
+              `Added ember-bootstrap configuration to '${BUILD_FILE}'.`
+            )
+          )
+        )
+        .catch(() => outputConfig(ui, config));
+    }
 
-        fs.writeJsonSync(bowerJSONPath, bowerJSON);
+    outputConfig(ui, config);
+  },
 
-        resolve();
-      } catch(error) {
-        reject(error);
+  _removeBowerPackages() {
+    let ui = this.ui;
+    let packages = ['bootstrap', 'bootstrap-sass'];
+    let dependencies = this.project.bowerDependencies();
+    let bowerJSONPath = resolve('bower.json');
+
+    let promises = packages.map(pkg => {
+      if (pkg in dependencies) {
+        ui.writeLine(
+          chalk.green(`  uninstall bower package ${chalk.white(pkg)}`)
+        );
+
+        return fs.readJson(bowerJSONPath).then(bowerJSON => {
+          delete bowerJSON.dependencies[pkg];
+          return fs.writeJson(bowerJSONPath, bowerJSON, { spaces: 2 });
+        });
       }
     });
+
+    return rsvp.all(promises);
   },
 
-  adjustBootstrapDependencies() {
-    let bootstrapVersion = this.bootstrapVersion;
-    let preprocessor = this.preprocessor;
+  _adjustBootstrapDependencies() {
+    let bs3Version = '^3.3.7';
+    let bs4Version = '^4.0.0';
```

Comment:
Can you please put them back as `const` at the module root, so it is easier to find them when we need to adjust!